app/src/main/java/org/oppia/android/app/topic/revisioncard/RevisionCardActivity.kt

```diff
@@ -2,22 +2,30 @@ package org.oppia.android.app.topic.revisioncard
 
 import android.content.Context
 import android.content.Intent
+import android.os.Build
 import android.os.Bundle
 import android.view.Menu
 import android.view.MenuItem
+import androidx.annotation.RequiresApi
+import androidx.core.content.res.ResourcesCompat
 import org.oppia.android.R
 import org.oppia.android.app.activity.ActivityComponentImpl
 import org.oppia.android.app.activity.InjectableAppCompatActivity
+import org.oppia.android.app.topic.RouteToRevisionCardListener
 import org.oppia.android.app.topic.conceptcard.ConceptCardListener
 import javax.inject.Inject
 
 /** Activity for revision card. */
 class RevisionCardActivity :
-  InjectableAppCompatActivity(), ReturnToTopicClickListener, ConceptCardListener {
+  InjectableAppCompatActivity(),
+  ReturnToTopicClickListener,
+  ConceptCardListener,
+  RouteToRevisionCardListener {
 
   @Inject
   lateinit var revisionCardActivityPresenter: RevisionCardActivityPresenter
 
+  @RequiresApi(Build.VERSION_CODES.LOLLIPOP)
```

Comment:
This implies there is code below that won't work on KitKat--what is it? Normally we need to get things behind API checks for this reason.

Reply:
continuing on the last review comment, it seems that changing colors is not possible below lollipop (see: https://stackoverflow.com/questions/33678266/changing-the-status-bar-color-on-android-kitkat)

Reply:
In that case, you need to move that bit of code to a method that uses ``@TargetApi``, and then conditionally call it using an if-check that checks the current build version. That way the app can still run on KitKat--it'll just bypass this extra logic.

That being said, what _will_ the behavior be on KitKat if we can't run the status bar color code? Also, do we need to change the status bar color? I feel like we should be doing whatever we do on other screens for this rather than deviating.

Reply:
Still partially unresolved, I think--can we remove the target annotation now?